src/ERC6538Registry.sol

```diff
@@ -9,13 +9,47 @@ contract ERC6538Registry is IERC6538Registry {
   /// @notice Maps a registrant's identifier to the scheme ID to the stealth meta-address.
   /// @dev `registrant` may be a standard 160-bit address or any other identifier.
   /// @dev `schemeId` is an integer identifier for the stealth address scheme.
-  mapping(bytes registrant => mapping(uint256 schemeId => bytes)) public stealthMetaAddressOf;
+  mapping(address registrant => mapping(uint256 schemeId => bytes stealthMetaAddress)) public
+    stealthMetaAddressOf;
+  mapping(address user => uint256 nonce) public nonceOf;
+
+  bytes32 private constant TYPE_HASH =
+    keccak256("EIP712Domain(string name,string version,uint256 chainId,address registryContract)");
```

Comment:
Similar comments to the ERC repo PR: 
- natspec for `nonceOf` and `TYPE_HASH`
- `public` visibility for `TYPE_HASH`
- method for users to increment their nonce

Reply:
Addressed them in the same commit as above https://github.com/ScopeLift/stealth-address-erc-contracts/pull/5/commits/4460b6edabe9487c5157a75f918ee7c116919931